docs/csharp/classes.md

```diff
@@ -57,25 +57,25 @@ When a class declares a base class, it inherits all the members of the base clas
   
 Unlike C++, a class in C# can only directly inherit from one base class. However, because a base class may itself inherit from another class, a class may indirectly inherit multiple base classes. Furthermore, a class can directly implement more than one interface. For more information, see [Interfaces](programming-guide/interfaces/index.md).  
   
-A class can be declared [abstract](https://msdn.microsoft.com/library/sf985hc5.aspx). An abstract class contains abstract methods that have a signature definition but no implementation. Abstract classes cannot be instantiated. They can only be used through derived classes that implement the abstract methods. By contrast, a [sealed](https://msdn.microsoft.com/library/88c54tsw.aspx) class does not allow other classes to derive from it. For more information, see [Abstract and sealed classes and class members](https://msdn.microsoft.com/library/ms173150.aspx).  
+A class can be declared [abstract](language-reference/keywords/abstract.md). An abstract class contains abstract methods that have a signature definition but no implementation. Abstract classes cannot be instantiated. They can only be used through derived classes that implement the abstract methods. By contrast, a [sealed](language-reference/keywords/sealed.md) class does not allow other classes to derive from it. For more information, see [Abstract and sealed classes and class members](programming-guide/classes-and-structs/abstract-and-sealed-classes-and-class-members.md).  
   
-Class definitions can be split between different source files. For more information, see [Partial class definitions](https://msdn.microsoft.com/library/wa80x488.aspx).  
+Class definitions can be split between different source files. For more information, see [Partial class definitions](programming-guide/classes-and-structs/partial-classes-and-methods.md).  
   
  
 ## Example
-In the following example, a public class that contains a single field, a method, and a special method called a constructor is defined. For more information, see [Constructors](https://msdn.microsoft.com/library/ace5hbzh.aspx). The class is then instantiated with the **new** keyword.
+In the following example, a public class that contains a single field, a method, and a special method called a constructor is defined. For more information, see [Constructors](programming-guide/classes-and-structs/constructors.md). The class is then instantiated with the **new** keyword.
 
 [!code-csharp[Class Example](../../samples/snippets/csharp/concepts/classes/class-example.cs)]  
   
 ## C# language specification  
-For more information, see the [C# language specification](https://msdn.microsoft.com/library/ms228593.aspx). The language specification is the definitive source for C# syntax and usage.
+For more information, see the [C# language specification](language-reference/language-specification/). The language specification is the definitive source for C# syntax and usage.
```

Comment:
missing file name index.md

Reply:
@mairaw I fixed all warnings. thanks for heads up.